### ISSUE TITLE
Find Widget doesn't trap focus

### ISSUE THREAD

**Issue by OlsonDev** [microsoft/vscode#2812]
The [latest updates](https://code.visualstudio.com/updates) mention the Find Widget's tab order is improved and I agree! Thanks!

However, unlike full Visual Studio, tabbing within the widget doesn't confine the focus; eventually I end up back in my document and the only way to get back is `Ctrl+F` or `Ctrl+H`. I'm not sure if this is intentional (with all your other a11y changes) but I generally prefer the way Visual Studio does it.

Bug? Setting?

**Comment by alexandrudima** [comment]
This is for accessibility purposes. In our environment the only way to go to the next focusable element is via Tab and for that reason we should not trap focus -- otherwise folks who cannot use a mouse will not be able to escape the find widget.

**Comment by OlsonDev** [comment]
I'm sorry, but that makes no sense. Every other IDE I've used traps focus in the find widget/panel.

I just tested with what I have installed:

- Sublime Text - `Esc` key escapes the find panels. `Tab` toggles between "Find What:" and "Replace With:"
- Visual Studio 2015 - `Esc` key escapes the find and replace panel. `Tab` toggles between all the fields.
- Atom - `Esc` key escapes both the `Find in current buffer` and `Find in project` panels. `Tab` toggles between the text fields.
- I could go on...

The `Esc` key should escape focus and restore focus to whatever was found.
